Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```
@@ -4189,6 +4189,21 @@ LoginStatusAPIEnabled:
     WebCore:
       default: false
 
+LoginStatusAPIRequiresWebAuthnEnabled:
+  type: bool
+  status: internal
+  category: dom
+  humanReadableName: "Require WebAuthn with the Login Status API"
+  humanReadableDescription: "Enable the proposed Login Status API which requires WebAuthn"
```

This preference doesn't actually enable the Login Status API, it just makes it so WebAuthn is required for it to work. So maybe this description should be something like:
```suggestion
  humanReadableDescription: "Require a recent WebAuthn authentication to set login status"
```